src/core/ext/filters/http/server/http_server_filter.cc

```diff
@@ -76,12 +73,6 @@ struct call_data {
   uint32_t* recv_initial_metadata_flags;
   bool seen_recv_initial_metadata_ready = false;
 
-  // State for intercepting recv_message.
```

Comment:
I think we also no longer need the `read_stream` or `have_read_stream` data members.

Reply:
fixed